spec/calculator.spec.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const Calculator = require('../calculator');
+const expect = require('chai').expect;
```

Comment:
We require in a module called "chai". This allows us to perform assertions (i.e. `expect(...).to.equal(...)`).